extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -497,6 +494,46 @@ export class AmpStoryPage extends AMP.BaseElement {
     ]);
   }
 
+  /** @override */
+  onMeasureChanged() {
+    if (!this.isFirstPage_) {
```

Comment:
Can you add a comment here with what you wrote on your description?

"Only measures from the first story page, that always gets built because of the prerendering optimizations we have (even with branching)"